### PR TITLE
Feature/correct license

### PR DESCRIPTION
**Pull request by sb43**
Hi Keiran,
Please check the changes to vafCorrect as discussed earlier.
1. Added prerelease.sh
2. Pulled License file form your branch
3. Added option to specify high depth regions file on command line
4. Updated README.md, setup.sh

Changes can be viewed by clicking:
https://github.com/cancerit/vafCorrect/compare/dev...feature/correct_license

### CODE REVIEW

File: perl/bin/cgpVaf.pl [hunk 1]
```diff
@@ -25,10 +22,10 @@
 ##########LICENCE##############################################################
 
 BEGIN {
-  use Cwd qw(abs_path);
-  use File::Basename;
-  $ENV{POSIXLY_CORRECT}=1;
-  unshift (@INC,dirname(abs_path($0)).'/../lib');
+  #use Cwd qw(abs_path);
```
Comment (keiranmraine) [inline perl/bin/cgpVaf.pl]: Code shouldn't be commented, if it works delete it.

File: .gitignore [hunk 2]
```diff
@@ -6,3 +6,7 @@ out*
 MYMETA*
 Makefile
 install_tmp
+perl/docs.tar.gz
```
Comment (keiranmraine) [inline .gitignore]: Perl docs should be committed, before a release it should be rebuilt via `prerelease.sh`

File: perl/bin/cgpVaf.pl [hunk 3]
```diff
@@ -1,7 +1,4 @@
-#!/software/perl-5.16.2/bin/perl -w
-
-eval 'exec /software/perl-5.16.2/bin/perl -w -S $0 ${1+"$@"}'
-    if 0; # not running under some shell
+#!/use/bin/perl
 
 ##########LICENCE############################################################
 # Copyright (c) 2016 Genome Research Ltd.
```
Comment (keiranmraine) [inline perl/bin/cgpVaf.pl]: You should update the date range on any file that you edit.  This should read 2016-2017

File: perl/prerelease.sh [hunk 4]
```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+##########LICENCE##########
+# Copyright (c) 2014 Genome Research Ltd.
```
Comment (keiranmraine) [inline perl/prerelease.sh]: Date must be corrected, if first instance of file just the year it was added.

**Comment by sb43** [comment]
Done!

**Comment by sb43** [comment]
@keiranmraine added docs to .gitignore file

**Pull request merged**